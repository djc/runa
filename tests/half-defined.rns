def main():
	if 0:
		a = 'x'
	print(a)
