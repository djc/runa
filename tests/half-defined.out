tests/half-defined.rns [4.8]: undefined name 'a'
    print(a)
          ^
